src/Workspaces/VisualBasic/Portable/CodeGeneration/VisualBasicSyntaxGenerator.vb

```diff
@@ -14,6 +14,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGeneration
 
 #Region "Expressions and Statements"
 
+        Public Overrides Function AddEventHandler([event] As SyntaxNode, handler As SyntaxNode) As SyntaxNode
+            Return SyntaxFactory.AddHandlerStatement(CType([event], ExpressionSyntax), CType(handler, ExpressionSyntax))
```

Comment:
I wasn't sure whether it was necessary for the C# case, although actually I think it's not, given that the assignment operators are the most associative operator and also right-associative.

For the VB case it's clear (right?) that no precedence issues can occur, so I thought to just leave it out.